pandunia-lekse.csv

```diff
@@ -2228,7 +2261,10 @@ reste|||fra:reste, eng:deu:Rest, pol:reszta, spa:por:resto|rest (remainder)|rest
 restoran||||restaurant||restaurante|||||||||||||restaŭracio|ravintola|restauracja
 restu||||remain (stay)||quedarse||||||||||||||jäädä jäljelle|zostać, zostawać
 retra||||return (go back)|retourner (revenir)|volver|voltar (retornar)|возвращаться||返回 (回来)||||लौटना|||||reiri|palata|wrócić, wracać, wycofać się, wycofywać się
-retre||||return (way back)||vuelta|||||||||||||reiro|paluu|
+retre||||back||dorso|||||||||||||||
```

Comment:
me fikra asli tarje si sahi. "rerurn" e "back of the body" si ali xey.

Reply:
aga "prede" mena "front", "bade" abla mena "back".

Reply:
si.